x-pack/plugins/security/public/management/_index.scss

```diff
@@ -0,0 +1,2 @@
+@import './edit_role/index';
+@import './edit_user/index';
```

Comment:
`index.scss` files shouldn't have any actual selectors in them. It should server solely as a manifest of imports. This `edit_user/index` file doesn't adhere to this pattern. It used to be called `_users.scss`

Reply:
Aha, I see, will fix, thanks!

Reply:
Fixed in https://github.com/elastic/kibana/pull/53880/commits/507b3faa5079f5686ef8873d58099ce2c8b67b6e